rosdep/python.yaml

```diff
@@ -6229,6 +6229,14 @@ python3-jsonschema:
   fedora: [python3-jsonschema]
   gentoo: [dev-python/jsonschema]
   ubuntu: [python3-jsonschema]
+python3-junitparser:
+  debian:
+    bullseye: [python3-junitparser]
+    buster: [python3-junitparser]
```

Comment:
@danthony06 why not applying the same logic as for `ubuntu`?

Reply:
Okay, switched from an allowed list of distributions to nulling stretch and allowing everything else.